# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-ft.h
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/cairo-ft.h
--- a/gfx/cairo/cairo/src/cairo-ft.h	Mon May 06 19:22:35 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-ft.h	Mon May 06 19:25:12 2024 +0000
@@ -41,7 +41,7 @@
 
 #if CAIRO_HAS_FT_FONT
 
-/* Fontconfig/Freetype platform-specific font interface */
+/* Fontconfig/FreeType platform-specific font interface */
 
 #include <ft2build.h>
 #include FT_FREETYPE_H
@@ -54,9 +54,7 @@
 
 cairo_public cairo_font_face_t *
 cairo_ft_font_face_create_for_ft_face (FT_Face         face,
-				       int             load_flags,
-				       unsigned int    synth_flags,
-				       void           *face_context);
+				       int             load_flags);
 
 /**
  * cairo_ft_synthesize_t: